docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/zendesk-source.md

```diff
@@ -0,0 +1,90 @@
+---
+id: zendesk-source
+title: Zendesk Source
+sidebar_label: Zendesk
+tags:
+  - cloud-to-cloud
+  - zendesk
+description: Learn how to collect audit logs using the Zendesk API.
+---
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/zendesk/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/zendesk/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/zendesk/example.tf';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/zendesk-icon.png')} alt="Zendesk-icon" width="70" />
+
+Zendesk is widely used across various industries, including technology, retail, healthcare, education, financial services, and many more. This platform is viral for its ease of use, scalability, and ability to integrate with other business tools and applications. The audit log shows various changes in the Zendesk account since the account was created. Records of these changes are saved indefinitely, and you can view the entire change history. This source collects the audit logs from the Zendesk API and ingests them into Sumo Logic.
```

Comment:
```suggestion
Zendesk is widely used across various industries, including technology, retail, healthcare, education, financial services, and many more. This platform is vital for its ease of use, scalability, and ability to integrate with other business tools and applications. The audit log shows various changes in the Zendesk account since the account was created. Records of these changes are saved indefinitely, and you can view the entire change history. This source collects the audit logs from the Zendesk API and ingests them into Sumo Logic.
```